actions/delete.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Default entity delete action
+ *
+ * @package Elgg
+ * @subpackage Core
```

Comment:
These annotations are not needed. There's in fact an issue about removing them from all files.